### DISCUSSION TITLE
Coinvolgimento scuole

### DISCUSSION THREAD

**Discussion by dennisangemi** [opendatasicilia/informa-covid19#41] [Idee/Proposte]
Nazzareno Prinzivalli nel gruppo [telegram di opendatasicilia](https://t.me/opendatasicilia/11868) scrive:

Sarebbe ipotizzabile lavorare assieme a studenti di una scuola superiore che si incaricano, dopo aver capito come fare, dell'aggiornamento dei dati? Io pensavo, valutandolo ovviamente assieme a voi, di creare una sinergia tra Comune e scuola/e: in tal modo il Comune si fa promotore di un'attività partecipata ma non chiede ai propri dipoendenti un "impegno" suppletivo" 😂
Di contro, gli studenti verrebero sollecitati sia sotto l'aspetto partecipativo che quello di interesse ai "dati"

**Comment by dennisangemi** [comment]
Sono contentissimo di leggere queste parole e credo sia necessario approfondire l'argomento.  Riuscire a coinvolgere gli/le studenti/esse delle scuole superiori sarebbe bellissimo ma, essendo una cosa a cui non avevamo pensato, sarebbe necessario concordare le modalità di coinvolgimento. 

- Come contattiamo le scuole? (C'è da preparare un testo)
- Come gestire le autorizzazioni dello sheet e del form? (il form, una volta creato è pubblico e chiunque abbia il link potrebbe inserire dati in dashboard)
- Coinvolgendo studenti/esse verrebbe meno la sicurezza in termini di veridicità del dato? (lo sheet potrà essere modificato)

Se decidiamo di coinvolgere le scuole, io proporrei di non affidare agli/lle studenti/sse il compito compito di effettuare data entry (che lasciamo al/la sindaco/a o dipendente comunale, per una sorta di ufficialità), piuttosto li inviterei a seguire il tutorial che stiamo realizzando per replicare il servizio e inviare il modulo con cui effettuare data entry al/la sindaco/a. In tal modo il comune non deve provare a comprendere il nostro tutorial e praticamente azzeriamo il rischio che il procedimento venga male eseguito. Al contempo preserviamo l'ufficialità essendo il comune ad inserire i dati. Che ne dite?

(Lo/a studente/ssa che repilicherà il servizio potrà comunque diffondere il link del modulo. Bisogna affidarsi al buon senso?)

**Reply by nazzaprinz** [reply]
Vado per ordine.
1) Le scuole possiamo contattarle tramite riferimenti personali diretti. Oppure, se esiste, utilizzare l'elenco delle scuole che hanno partecipato ad ASOC. Oppure ancora, se si contatta prima il sindaco o chi per esso, potrebbe avere lui interesse a segnalare un istituto. Oppure con tutti i metodi :-)

2) Ma non sarebbe possibile differenziare l'accesso al form per il data entry da quello relativo alla visualizzazione dei dati inseriti?

3) In caso di risposta affermativa alla 2) il problema dovrebbe venir meno, ritengo.

Dal punto di vista dell'ufficialità perchè non si potrebbe pensare ad una convenzione tra sindaco e scuola incaricata? I dati di input verrebbero comunicati sia alla scuola che al Comune: la prima inserisce, il secondo supervisiona.
